doc/General/Updating.md

```diff
@@ -23,7 +23,7 @@ you can do so by running the following commands:
 cd /opt/librenms
 git pull
 ./scripts/composer_wrapper.php install --no-dev
-./build-base.php
+php includes/sql-schema/update.php
```

Comment:
oh, I guess people could still be on an age old version, but at least we could remove the. build-base.php script.

Reply:
I'll do some following up cleanup later